Task5-2/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+    module:{
+        rules:[
+            {
+                test: /\.html$/,
+                use:[
+                    {
+                        loader: "html-loader",
+                        options: {minimize : true}
+                    }
+                ]
+            },
+            {
+                test: /\.(png|jpg)$/,
+                use:[
+                    'file-loader'
+                ]
+            },
+            {
+                test: /\.scss$/,
+                use: [
+                    "style-loader",
```

Comment:
You use `style-loader` and all CSS styles are injected directly inside HTML which is really bad. You should locate all CSS inside `main.css` file and load only this file.

Reply:
Added settings to webpack.config